site/en/blog/show-picker/index.md

```diff
@@ -0,0 +1,214 @@
+---
+layout: 'layouts/blog-post.njk'
+title: 'Show a browser picker for date, time, color, and files'
+description: >
+  The web platform now ships with a canonical way to show a browser picker.
+authors:
+  - beaufortfrancois
+date: 2022-01-14
+hero: image/vvhSqZboQoZZN9wBvoXq72wzGAf1/zWT5kk3Q5RlDNqGkdgis.jpeg
+alt: A photo of a monthly schedule printed on white paper
+tags:
+  - chrome-99
+---
+
+For a long time, you had to resort to custom widget libraries or hacks to show a
+date picker. The web platform now ships with `HTMLInputElement` `showPicker()`,
```

Comment:
```suggestion
date picker. The web platform now ships with `HTMLInputElement.showPicker()`,
```

Reply:
@jpmedley We decided to not use `HTMLInputElement.showPicker()` eventually. See discussion at https://github.com/GoogleChrome/developer.chrome.com/pull/1934/files/ac307daf5c37088e210cfa0d42c9e748acee17a4#r784180549

Reply:
We need to be consistent with the rest of the site. This is how it's done.

Reply:
Do you have other examples where it's done this way in dcc?

Reply:
I'd need to look, but this is the way I've been doing for 25 years on every site and document I've worked on.

Reply:
I found an article you reviewed where I use "_The NDEFReader `scan()` and `write()` methods_", not "_`NDEFReader.scan()` and `NDEFReader.write()` methods_". See https://web.dev/nfc/#:~:text=The%20NDEFReader%20scan()%20and%20write()%20methods

Shall we use "_HTMLInputElement `showPicker()`_" instead of "_`HTMLInputElement` `showPicker()`_" then?

Reply:
> We need to be consistent with the rest of the site. This is how it's done.

The rest of the site being misleading and poorly-edited in this regard is not a reason to propagate that confusion forward.

Reply:
This is not the place to discuss standards. If you feel strongly about it, [open a ticket](https://github.com/GoogleChrome/web.dev/issues) and the writing team will discuss it.

Reply:
@jpmedley What did you think of my suggestion at https://github.com/GoogleChrome/developer.chrome.com/pull/1934#discussion_r791809226?

Reply:
Done: https://github.com/GoogleChrome/web.dev/issues/7249

Reply:
I changed my suggestion. In replacing a period with a space, I have to infer that this is an instance method. If you need to make a distinction then you need to say it.